streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -17,116 +17,109 @@
 
     /**
-     * Creates a new instance of KStream consists all elements of this stream which do not satisfy a predicate
+     * Create a new instance of KStream consists all elements of this stream which do not satisfy a predicate.
      *
-     * @param predicate the instance of Predicate
-     * @return the instance of KStream with only those elements that do not satisfy the predicate
+     * @param predicate     the instance of {@link Predicate}
      */
     KStream<K, V> filterOut(Predicate<K, V> predicate);
 
     /**
-     * Creates a new instance of KStream by applying transforming each element in this stream into a different element in the new stream.
+     * Create a new instance of {@link KStream} by transforming each element in this stream into a different element in the new stream.
      *
-     * @param mapper the instance of KeyValueMapper
-     * @param <K1>   the key type of the new stream
-     * @param <V1>   the value type of the new stream
-     * @return the instance of KStream
+     * @param mapper        the instance of {@link KeyValueMapper}
+     * @param <K1>          the key type of the new stream
+     * @param <V1>          the value type of the new stream
      */
     <K1, V1> KStream<K1, V1> map(KeyValueMapper<K, V, KeyValue<K1, V1>> mapper);
 
     /**
-     * Creates a new instance of KStream by transforming each value in this stream into a different value in the new stream.
+     * Create a new instance of {@link KStream} by transforming each value in this stream into a different value in the new stream.
      *
-     * @param mapper the instance of ValueMapper
-     * @param <V1>   the value type of the new stream
-     * @return the instance of KStream
+     * @param mapper        the instance of {@link ValueMapper}
+     * @param <V1>          the value type of the new stream
      */
     <V1> KStream<K, V1> mapValues(ValueMapper<V, V1> mapper);
 
     /**
-     * Creates a new instance of KStream by transforming each element in this stream into zero or more elements in the new stream.
+     * Create a new instance of {@link KStream} by transforming each element in this stream into zero or more elements in the new stream.
      *
-     * @param mapper the instance of KeyValueMapper
-     * @param <K1>   the key type of the new stream
-     * @param <V1>   the value type of the new stream
-     * @return the instance of KStream
+     * @param mapper        the instance of {@link KeyValueMapper}
+     * @param <K1>          the key type of the new stream
+     * @param <V1>          the value type of the new stream
      */
     <K1, V1> KStream<K1, V1> flatMap(KeyValueMapper<K, V, Iterable<KeyValue<K1, V1>>> mapper);
 
     /**
-     * Creates a new stream by transforming each value in this stream into zero or more values in the new stream.
+     * Create a new instance of {@link KStream} by transforming each value in this stream into zero or more values in the new stream.
```

Comment:
Perhaps clarify to: "by transforming each element in this stream into zero or more elements with the same key but with new values."